code/modules/cargo/bounties/mech.dm

```diff
@@ -15,22 +15,22 @@
 
 /datum/bounty/item/mech/ripley
 	name = "APLU \"Ripley\""
-	reward = 13000
+	reward = 2500
 	wanted_types = list(/obj/mecha/working/ripley)
 
 /datum/bounty/item/mech/odysseus
 	name = "Odysseus"
-	reward = 11000
+	reward = 2700
 	wanted_types = list(/obj/mecha/medical/odysseus)
 
 /datum/bounty/item/mech/gygax
 	name = "Gygax"
-	reward = 28000
+	reward = 4500
```

Comment:
according to ED 209 prices, this is 4.5 of ED 209. Yet making ED 209 is extremely easy and would not be of much of cost of materials. Gygax on the other hand uses diamonds and lots of materials plus research components, etc. And Labor too. This is underpriced

Reply:
PS. Diamond drill cost 2000, which means that just diamonds along would be 2000 in creating Gygax